src/helm/benchmark/metrics/bhasa_metrics.py

```diff
@@ -0,0 +1,316 @@
+            preds = [completion.text.strip() for completion in sorted_completions]
+
+            for name, metric in self.metrics.items():
+                name = MetricName(name)
+                metric = cast(Callable[[str, str], float], metric)
+                score_1 = max(metric(gold.output.text.strip(), preds[0]) for gold in golds)
+                score_k = max(metric(gold.output.text.strip(), pred) for gold in golds for pred in preds)
+
+                metrics = [Stat(name).add(score_1)]
+                if adapter_spec.num_outputs != 1:
+                    metrics.append(Stat(replace(name, name=f"{name.name}@{adapter_spec.num_outputs}")).add(score_k))
+                stats.extend(metrics)
+
+        return stats
+
+class BhasaSummarizationMetric(Metric):
+    """Summarization Metrics
+
+    This class computes the following standard summarization metrics
+
+    1. XL-Sum Rouge-L (F1 score, using the "mid" result when performing bootstrap aggregation)
+
+    @inproceedings{hasan-etal-2021-xl,
+    title = "{XL}-Sum: Large-Scale Multilingual Abstractive Summarization for 44 Languages",
+        author = "Hasan, Tahmid  and
+            Bhattacharjee, Abhik  and
+            Islam, Md. Saiful  and
+            Mubasshir, Kazi  and
+            Li, Yuan-Fang  and
+            Kang, Yong-Bin  and
+            Rahman, M. Sohel  and
+            Shahriyar, Rifat",
+        editor = "Zong, Chengqing  and
+            Xia, Fei  and
+            Li, Wenjie  and
+            Navigli, Roberto",
+        booktitle = "Findings of the Association for Computational Linguistics: ACL-IJCNLP 2021",
+        month = aug,
+        year = "2021",
+        address = "Online",
+        publisher = "Association for Computational Linguistics",
+        url = "https://aclanthology.org/2021.findings-acl.413",
+        doi = "10.18653/v1/2021.findings-acl.413",
+        pages = "4693--4703",
+        github = "https://github.com/csebuetnlp/xl-sum",
+    }
+    
+    """
+
+    def __init__(self, language: str = 'en'):
+        self.language: str = language
+        self.rouge_metrics = {
+            "rougeL": "xlsum_rouge_l",
+        }
+        self.rouge_scorer = self._get_bhasa_rouge_scorer(self.rouge_metrics)
+
+    def _get_bhasa_rouge_scorer(self, rouge_metrics: str) -> Callable[[str, str], float]:
+        language = "thai" if self.language == "th" else None
+        return rouge_scorer.RougeScorer(list(rouge_metrics.keys()), use_stemmer=False, lang=language)
+
+    def _compute_rouge(self, refs: List[str], pred: str) -> Dict[str, float]:
+        metrics: Dict[str, float] = {}
+
+        aggregator = BootstrapAggregator()
+        for ref in refs:
+            aggregator.add_scores(self.rouge_scorer.score(ref, pred))
+        aggregates = aggregator.aggregate()
+        for key, value in self.rouge_metrics.items():
+            metrics[value] = aggregates[key].mid.fmeasure * 100
```

Comment:
Is this to make the range 0 to 100 instead of 0 to 1? We generally prefer the 0 to 1 range.